frontend/src/api/appwriteApi.js

```diff
@@ -127,6 +126,37 @@ let api = {
 		return api.provider().database.deleteDocument(collectionId, documentId);
 	},
 
+	createAnnouncement: (data) => {
+		console.log(data);
+		return api
+			.provider()
+			.database.createDocument(
+				AppwriteServer.announcementCollectionID, 
+				data, 
+				["*"], 				// read permission
+				["team:Admins"]		// write permission
+			);		
+	},
```

Comment:
Here you probably want this: 
```suggestion
	createAnnouncement: async (data) => {
		console.log(data);
		return api
			.provider()
			.database.createDocument(
				AppwriteServer.announcementCollectionID,
				data,
				["*"], 				// read permission
				["team:" + await api.getTeamId("Admins")]		// write permission
			);
	},
```

If we say "team:Admins", then the team with team ID "Admins" will have write access, and not the actual Admins team

Reply:
real admins are recognizable by the 'owner' role

Reply:
While this is true, I think announcements should be creatable/editable/deletable by every admin.